src/main/java/productDependency/CreateOrderPage.java

```diff
@@ -0,0 +1,1481 @@
+ 			JavaScriptExec.sleep();
+
+ 		
+
+	}
+	
+ 		@FindBy(how = How.XPATH, using = "//a//following::span[text()='Update'][2]")
+ 		private WebElement clickUpdateButton1;
+
+ 		/**
+ 		 * Method to click on Update button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
+ 		public void clickUpdateButton1() throws IOException {
+ 			CreateOrderPage sp = new CreateOrderPage(driver);
+ 			log.info("click on Update button.");
+ 			Assert.assertTrue(clickUpdateButton.isDisplayed());
+ 			clickUpdateButton.click();
+ 			JavaScriptExec.sleep();
+
+ 		
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//a[@class='submit save']//*[text()='Save Changes']")
+	private WebElement clickSaveChangesButton;
+
+	/**
+	 * Method to click on Save Changes Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickSaveChangesButton() throws IOException {
+		OrderHierarchiesPage sp = new OrderHierarchiesPage(driver);
+		log.info("Click on Save Changes Button");
+		Assert.assertTrue(clickSaveChangesButton.isDisplayed());
+		JavaScriptExec.sleep();
+		clickSaveChangesButton.click();
+
+	}
+	
+	@FindBy(how = How.XPATH, using = "//*[@id='order-18257']/td[1]")
+	//*[@id="order-18257"]/td[1]
+	//*[@id="18258"]/span
+	private WebElement clickOrder;
+
+	/**
+	 * Method to click on Save Changes Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickOrder() throws IOException {
+		OrderHierarchiesPage sp = new OrderHierarchiesPage(driver);
+		log.info("Click on Save Changes Button");
+		Assert.assertTrue(clickOrder.isDisplayed());
+		clickOrder.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//a[@class='submit order']//*[text()='Generate Invoice']")
+	private WebElement clickGenerateInvoiceButton;
+
+	/**
+	 * Method to Click on Generate Invoice Button.
+	 * 
+	 * @throws IOException
+	 */
+	public void clickGenerateInvoiceButton() throws IOException {
+		JavaScriptExec.scrollToElementOnPage(driver, clickGenerateInvoiceButton);
+		GeneratingInvoicePage sp = new GeneratingInvoicePage(driver);
+		log.info("Click on Generate Invoice Button");
+		Assert.assertTrue(clickGenerateInvoiceButton.isDisplayed());
+		clickGenerateInvoiceButton.click();
+
+	}
+
+	@FindBy(how = How.XPATH, using = "//div[@class='msg-box successfully']//*[text()='Done']")
+	private WebElement verifyConfirmationMsg;
+
+	/**
+	 * Method to verify Account Type is created Successfully.
+	 * 
+	 * @throws IOException
+	 */
+	public void verifyConfirmationMsg() throws IOException {
+		CreateOrderPage sp = new CreateOrderPage(driver);
+		log.info("Verifying if Account Type is created Successfully or not");
+		JavaScriptExec.sleep();
+		Assert.assertTrue(verifyConfirmationMsg.isDisplayed(),
+				"Assert Failed as its unable to search text in Logged in Page");
+	}
+	
+	
+ 		@FindBy(how = How.XPATH, using = "//button[@id='newSubOrder']")
+ 		
+ 	private WebElement clickNewSubOrderButton;
+
+ 		/**
+ 		 * Method to click on New Sub Order Order button.
+ 		 * 
+ 		 * @throws IOException
+ 		 */
```

Comment:
@mayankchhajer Please format  the code properly by using Ctrl+Shift+F

Reply:
Done